cellprofiler/modules/enhanceorsuppressfeatures.py

```diff
@@ -1,7 +1,10 @@
 # coding=utf-8
 
 """
-**Enhance Or Suppress Features** enhances or suppresses certain image
+EnhanceOrSuppressFeatures
+=========================
+
+**EnhanceOrSuppressFeatures** enhances or suppresses certain image
```

Comment:
Can you add a sentence to indicate it's 3D compatible?  Also, are all settings 3D compatible or only some?